src/index.d.ts

```diff
@@ -911,6 +915,29 @@ export interface SourceEditor {
 	show_gutter?: boolean;
 }
 
+export interface CommitTemplate {
+	/**
+	 * Used to identify a commit template when multiple commit templates are available.
+	 */
+	label?: string;
+	/**
+	 * Set the string for the commit template. This will only be used if template_path is not set.
+	 */
+	template_string?: string;
+	/**
+	 * Sets the path for a file containing your commit template. The file path should be relative to the root directory.
+	 */
+	template_path?: string;
+	/**
+	 * Define inputs used to populate data placeholders in the commit template.
+	 */
+	_inputs?: Record<string, Input>;
```

Comment:
Does this support _structures, _select_data etc as well?

Reply:
It doesn't! But certainly should in the future!